webpack.config.js

```diff
@@ -170,6 +170,19 @@ module.exports = [
             library: 'JitsiMeetExternalAPI',
             libraryTarget: 'umd'
         })
+    }),
+
+    // The Webpack configuration to bundle external_api.js (aka
+    // JitsiMeetExternalAPI).
+    Object.assign({}, config, {
+        entry: {
+            'browser_capabilities':
+                './react/features/browser-capabilities/index.js'
+        },
+        output: Object.assign({}, config.output, {
+            library: 'BrowserCapabilities',
+            libraryTarget: 'umd'
+        })
```

Comment:
What's the difference between defining the bundle in index 2 (line 177) of the exported array versus within index 0 (line 138)?

Reply:
I was not sure how to specify:
```
output: Object.assign({}, config.output, {
            library: 'BrowserCapabilities',
            libraryTarget: 'umd'
        })
```

Also I think there was something that webpack will reuse some cached stuff and build faster if everything is declared within a single configuration. But I'm not sure about it and in our case we don't have shared code with anything. 

Anyway do you want me to try to put this configuration within index 0?

Reply:
If there was no explicit reason for putting it in a separate index (index 2) then I do recommend trying it out in index 0 for consistency. If there was a specific reason to put it in a separate index, then we should have a (quick) discussion of when to put define a new bundle in a new index and when not to.

Reply:
Any idea how to put it in index 0?

Reply:
I see 2 options here:
- The current one. Note that in the current solution we have:
```
output: Object.assign({}, config.output, {
            library: 'BrowserCapabilities',
            libraryTarget: 'umd'
        })
```
This will attach BrowserCapabilities to the window object.
 - If we add it to index 0 we can't specify `output`. So we'll have to manually attach BrowserCapabilities to the window object.

About the consistency argument we already have the definition of external_api which is exactly like the browser_capabilities one. So I don't think we are adding inconsistency here.

But I don't mind changing it so let's discuss it offline! If we change it, probably make sense to put external_api in index 0 too.